### PR TITLE
Expose SingleHotel to AppRegistry

### LINKED ISSUE #458: Add ability to open single hotel
We should expose (?) new component to create a possibility to open a single hotel from anywhere (not the search but one hotel directly).

### PR DESCRIPTION
**Pull request by tbergq**
closes #458

### CODE REVIEW

File: ios/reactNativeApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1177,17 +1177,7 @@
...
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/EvilIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Feather.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/FontAwesome.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Foundation.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Ionicons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/MaterialCommunityIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/MaterialIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Octicons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/SimpleLineIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Zocial.ttf",
```
Comment (mrtnzlml) [inline ios/reactNativeApp.xcodeproj/project.pbxproj]: Why again? :) I am so confused by this...

Comment (tbergq) [reply ios/reactNativeApp.xcodeproj/project.pbxproj]: Wish I knew 🙈 Maybe we can ask our iOS developer. Maybe he knows why

File: app/hotels/README.md [hunk 2]
```diff
@@ -17,3 +17,26 @@ const render = () => <HotelsStandalonePackage
...
+
+const render = () => <SingleHotelStandAlonePackage
+  bookingComAffiliate="123456"
+  language="en"
+  currency="EUR"
+  onBackClicked={() => goToHomepage()}
+  dataSaverEnabled={false}
+  checkin="2018-05-01" // Any valid moment date format
+  checkout="2018-05-05" // Any valid moment date format 
+  roomsConfiguration={roomsConfiguration}
```
[the remark this answers is not in the document]
Comment (tbergq) [reply app/hotels/README.md]: Good catch, I will add it

**Comment by tbergq** [comment]
@mrtnzlml please check again

**Pull request merged**